### PR TITLE
Profile screen schedule

### PR DESCRIPTION
**Pull request by NelsonTanCS**
- add schedule component to profile screen

- select day color is darker

### CODE REVIEW

File: api/server/dayTimeModel.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+var mongoose = require("mongoose");
```
Comment (apratan) [inline api/server/dayTimeModel.js:1]: Remove this model? We decided to not use this anymore, right?

File: frontend/screens/ProfileScreen/index.js [hunk 2]
```diff
@@ -6,16 +6,81 @@ import {
...
+import ScheduleComponent from '../../components/ScheduleComponent/index'
+import Day from '../../enums/Day';
+import moment from 'moment';
+import Schedule from '../../models/Schedule';
 
 function Profile({ navigation }) {
   const [user, setUser] = useState(JSON.parse('{"username" : "", "email" : "", "displayName" : ""}'));
 
+  const schedule = new Schedule();
+  schedule.availability = [
```
Comment (apratan) [inline frontend/screens/ProfileScreen/index.js:18]: This is just dummy data, right?

Comment (NelsonTanCS) [reply frontend/screens/ProfileScreen/index.js:18]: yeaah

File: api/server/dayTimeModel.js (hunk 1 above)
Comment (NelsonTanCS) [reply api/server/dayTimeModel.js]: done

File: frontend/screens/ProfileScreen/index.js (hunk 2 above)
Comment (bfok123) [reply frontend/screens/ProfileScreen/index.js:18]: Yes this is dummy data, and I think we are removing the Schedule model as well. I'm going to add in controllers and such for adding things to a user's schedule then you can use it to pull the schedule from the database. I changed my schedule component for these changes in a different branch. You can push the dummy data for now then change them later once I'm done with the controllers.

**Pull request merged**